bundle/manifests/quay-operator.clusterserviceversion.yaml

```diff
@@ -187,13 +187,13 @@ spec:
                     fieldRef:
                       fieldPath: metadata.annotations['quay-version']
                 - name: RELATED_IMAGE_COMPONENT_QUAY
-                  value: registry-proxy.engineering.redhat.com/rh-osbs/quay-quay-rhel8:v3.5.7
+                  value: registry-proxy.engineering.redhat.com/rh-osbs/quay-quay-rhel8:v3.5.8
                 - name: RELATED_IMAGE_COMPONENT_CLAIR
-                  value: registry-proxy.engineering.redhat.com/rh-osbs/quay-clair-rhel8:v3.5.7
+                  value: registry-proxy.engineering.redhat.com/rh-osbs/quay-clair-rhel8:v3.5.8
                 - name: RELATED_IMAGE_COMPONENT_BUILDER
-                  value: registry-proxy.engineering.redhat.com/rh-osbs/quay-quay-builder-rhel8:v3.5.7
+                  value: registry-proxy.engineering.redhat.com/rh-osbs/quay-quay-builder-rhel8:v3.5.8
                 - name: RELATED_IMAGE_COMPONENT_BUILDER_QEMU
-                  value: registry-proxy.engineering.redhat.com/rh-osbs/quay-quay-builder-qemu-rhcos-rhel8:v3.5.7
+                  value: registry-proxy.engineering.redhat.com/rh-osbs/quay-quay-builder-qemu-rhcos-rhel8:v3.5.8
```

Comment:
dito

Reply:
I'll replace these as well, but it won't work out of the box since we need to use digests here right?

Reply:
![istockphoto-171257665-612x612](https://user-images.githubusercontent.com/1385436/134684629-e048d1e7-a0bc-45e1-bc3a-8ec4f05fee57.jpg)

Reply:
😭 😂